SCClassLibrary/Common/Audio/Splay.sc

```diff
@@ -1,18 +1,33 @@
+LevelComp {
+	*new { |levelComp, rate, n|
+		if (levelComp == true) {
+			^if(rate == \audio) {
+				// ar default is equal power for backwards compatibility
+				n.reciprocal.sqrt
+			} {
+				// kr default is equal amplitude
+				n.reciprocal
+			}
+		};
+		// if false, level is untouched
+		if (levelComp == false) { ^1 };
+
+		// if a number or control signal,
```

Comment:
you could also want to scale an audio signal with a float levelComp.

Reply:
```level``` is used to scale the output signal by a float by hand;
the ```levelComp``` input expects an exponent for the ```(1/n) ** levelComp```formula.

Reply:
Sorry, I meant to suggest to drop the comment "// if a number or control signal,".

Reply:
OK, still not sure I understand what you propose 
- I thought it makes sense to keep comments that explain:
first we test for booleans for compatibility, and if we get past that, 
we know we have a SimpleNumber or OutputProxy / control signal, 
which become the exponent for the formula. 
what would you prefer here?

Reply:
well, I'd explain it as follows:
- first, we check if the levelComp is one of the two boolean values.
- if not, we use the formula – at whatever rate the numbers/ugens are (they may be audio rate)

Reply:
I just noted that the return (`^(n.reciprocal ** levelComp.clip(0.0, 1.0))`) is missing.

Reply:
missing return was already fixed, 
and ok, understood it now, changed it to: 
```	
// now levelComp is a number or UGen,
// so we scale by exponent: ... 
```